### DISCUSSION TITLE
[Feature] `Appears in` for mobile apps

### DISCUSSION THREAD

**Discussion by maekawatoshiki** [immich-app/immich#13071] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

On the web app, which albums a picture is placed in are shown in "Appears in" section.
On the mobile app (particularly I'm using Android one), however, there is no "Appears in" in picture info.

It would be helpful to add the feature for mobile apps.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#11615
